packages/nextra-theme-docs/src/index.tsx

```diff
@@ -122,13 +122,7 @@ const Body = ({
   }
 
   return (
-    <article
-      className={cn(
-        'nextra-body relative flex w-full min-w-0 max-w-full justify-center pb-8 pr-[calc(env(safe-area-inset-right)-1.5rem)]',
-        themeContext.typesetting &&
-          `nextra-body-typesetting-${themeContext.typesetting}`
```

Comment:
Typesetting is used here: https://github.com/shuding/nextra/blob/core/examples/swr-site/pages/meta.en-US.json#L49

Reply:
I searched for classes that might start with `nextra-body-typesetting` and didn't find any, does it means that it will be added in the future?
<img width="633" alt="image" src="https://user-images.githubusercontent.com/7361780/183918985-22164008-aede-4495-a647-cc0278c0cef3.png">

Reply:
bringed back